sys-libs/llvm-libunwind/llvm-libunwind-3.8.1.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+DESCRIPTION="C++ runtime stack unwinder from LLVM"
+HOMEPAGE="https://github.com/llvm-mirror/libunwind"
+if [ "${PV%9999}" = "${PV}" ] ; then
```

Comment:
I thought ${PN}-9999.ebuild serves as kind of a template ebuild for its package. No?